contentcuration/contentcuration/api.py

```diff
@@ -3,10 +3,12 @@
 It exposes several convenience functions for accessing content
```

Comment:
This docstring doesn't seem so accurate anymore.

Could also be useful to separate out the ricecooker endpoints into a separate API file, especially since they'll have different logic for things like permissions (token-based, rather than session-based, etc).